docs/build/tutorials/how-to/deploy-contract.md

```diff
@@ -0,0 +1,480 @@
+
+### Installation
+
+In the root directory of your project, execute this command:
+
+:::code-tabs
+
+@tab yarn
+
+```bash
+yarn add -D @matterlabs/hardhat-zksync-ethers hardhat zksync-ethers ethers
+```
+
+@tab npm
+
+```bash
+npm i -D @matterlabs/hardhat-zksync-ethers
+```
+
+:::
+
+Add the plugin at the top of the **hardhat.config.ts** file:
+
+```bash
+import "@matterlabs/hardhat-zksync-ethers";
+```
+
+### Configuration
+
+To deploy contracts with **hardhat-zksync-ethers**, use the similar configuration for our **hardhat.config.ts** as shown [here.](deploy-contract.md#configuration)
+The **accounts** section enables to specify wallet private keys which help us deploy contracts with automatically populated wallet within **hardhat-zksync-ethers** plugin, but it can still be manually created for other use cases.
+
+```bash
+{
+    url: 'https://sepolia.era.zksync.dev', // you should use the URL of the zkSync network RPC
+    ethNetwork: 'sepolia',
+    zksync: true,
+    accounts:["0xYOUR_ACCOUNT_PRIVATE_KEY"] // put your private key here
+},
+```
+
+### Deployment process
+
+For the following examples, we use same **SimpleStorage.sol** smart contract as shown [here](deploy-contract.md#create-a-new-smart-contract).
+
+### Compilation
+
+We also use the same steps as shown above in this [section.](deploy-contract.md#compilation)
+
+### Deploy scripts
+
+In this section, we'll create several deployment scripts to demonstrate various methods of deploying a smart contract.
+Since we've placed the private key inside the `account` section of **hardhat.config.ts**, we're able to automatically connect the default wallet to the network.
+
+:::note hardhat-zksync-ethers does not require deploy folder
+
+These plugins don't use the **hardhat-zksync-deploy** plugin, so the `deploy` folder isn't used for deployments. Instead, scripts will be stored within the `scripts` folder and executed using the `hardhat run scripts/SCRIPT_NAME` command.
+:::
+
+::: note Usage of getWallet helper method
+
+Since we aim to deploy contracts to the `zkSync Sepolia Testnet`, and considering we've configured an `account` section inside the network settings in the **hardhat.config.ts**, we can automatically connect the default wallet to the network. All helper methods for deployments will utilize this approach under the hood, so users don't need to instantiate a new wallet object.
+
+If we want to obtain a wallet instance for a new private key that is not specified in the `account` section, we can do so with:
+
+```javascript
+const wallet = await hre.zksyncEthers.getWallet(PRIVATE_KEY);
+```
+
+If we extend the `account` section inside the `network` settings in the **hardhat.config.ts** with more than one account, we can retrieve the wallets by indexes:
+
+```bash
+....
+accounts: ["0xYOUR_ACCOUNT_PRIVATE_KEY_1", "0xYOUR_ACCOUNT_PRIVATE_KEY_2", "0xYOUR_ACCOUNT_PRIVATE_KEY_3"]
```

Comment:
Similar to above comment about not including Private Keys in files that are easily exposable